taichi/rhi/cuda/cuda_driver.cpp

```diff
@@ -79,4 +62,63 @@ CUDADriver &CUDADriver::get_instance() {
   return get_instance_without_context();
 }
 
+CUDADriverBase::CUDADriverBase() {
+  disabled_by_env_ = (get_environ_config("TI_ENABLE_CUDA", 1) == 0);
```

Comment:
This variable should be used for early exits in `CUDADriver()`, `CUSPARSEDriver()`, ... Otherwise it is useless.

Reply:
`disabled_by_env_` is used to check `is_cuda_api_available`  when the `arch=ti.cuda`.